src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -81,7 +81,7 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       pattern: ['#B58FC1'],
     },
     point: {
-      r: 5,
+      r: 3,
```

Comment:
What is this change in parameter about?

Reply:
It's the radius of the point on the graph, it just got smaller